src/signals/settings/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Route } from 'react-router-dom';
+
+import { isAuthenticated } from 'shared/services/auth/auth';
+
+import LoginPage from 'components/LoginPage';
+
+import UsersOverviewContainer from './users/containers/Overview';
+
+export const SettingsModule = ({ match: { url } }) =>
+  !isAuthenticated() ? (
+    <Route component={LoginPage} />
```

Comment:
please put these with the other Routes in IncidentManagementModuleComponent.
better in 1 place I think

Reply:
I don't agree; routes can be nested so that URL parts can be inherited. That way we also separate concerns.

Reply:
it makes it more difficult to put it not with the other  routes in the admin